fclist.py

```diff
@@ -126,23 +121,29 @@ def build_fc_key_table():
     ]
 
     for key in bool_keys:
-        keys[key] = (ffi.new('char[]', key), get_bool,)
+        key_in_bytes = str.encode(key)
+        keys[key] = (ffi.new('char[]', key_in_bytes), get_bool,)
     for key in int_keys:
-        keys[key] = (ffi.new('char[]', key), get_int,)
+        key_in_bytes = str.encode(key)
+        keys[key] = (ffi.new('char[]', key_in_bytes), get_int,)
     for key in double_keys:
-        keys[key] = (ffi.new('char[]', key), get_double,)
+        key_in_bytes = str.encode(key)
+        keys[key] = (ffi.new('char[]', key_in_bytes), get_double,)
     for key in string_keys:
-        keys[key] = (ffi.new('char[]', key), get_string,)
+        key_in_bytes = str.encode(key)
+        keys[key] = (ffi.new('char[]', key_in_bytes), get_string,)
```

Comment:
Replace by `keys[key] = (ffi.new('char[]', str.encode(key)), get_string,)` for this and the above loops